src/data/finset/basic.lean

```diff
@@ -2694,6 +2694,11 @@ lemma exists_list_nodup_eq [decidable_eq α] (s : finset α) :
   ∃ (l : list α), l.nodup ∧ l.to_finset = s :=
 ⟨s.to_list, s.nodup_to_list, s.to_list_to_finset⟩
 
+lemma to_list_insert [decidable_eq α] {a : α} {s : finset α} (h : a ∉ s) :
+  a :: s.to_list ~ (insert a s).to_list :=
```

Comment:
I think this would make more sense in reverse, since it's the RHS of the lemma that reads `to_list (insert ...)` right now.

Reply:
true! editing